README.md

```diff
@@ -14,9 +14,9 @@ npm install -g csv2influx
 ## Usage
 
 ```
-csv2influx init        Creates template config file
-csv2influx data.csv    Loads config file from current directory
-                       then imports file data.csv to your influx
+csv2influx init                                       Creates template config file
+csv2influx --config path/to/config.json data.csv      Loads config from path/to/config.json 
```

Comment:
csv2influx.conf.json 
I don't want to be forced to put `--config` 
for example, when you use `npm install` , then npm tries to use `package.json` from current directory. Same thing here. On top of this, it breaks our working example. 

Btw, we call it `csv2influx.conf.json` , but not `config.json`

But he idea is cool.